test/app.test.js

```diff
@@ -613,6 +613,7 @@ describe('app', function() {
     var app, db;
     beforeEach(function() {
       app = loopback();
+      app.set('remoting', { errorHandler: { debug: false, log: false }});
```

Comment:
I think it's better to set `debug:true` in tests, so that we get more details in error messages. Thoughts?